src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -372,7 +373,8 @@ export class BundleDataClient {
         if (
           fill.blockNumber < blockRanges[chainIndex][0] ||
           fill.blockNumber > blockRanges[chainIndex][1] ||
-          isZeroValueFillOrSlowFillRequest(fill)
+          isZeroValueFillOrSlowFillRequest(fill) ||
+          invalidOutputToken(fill)
```

Comment:
What's the intention with this check? `invalidOutputToken()` is only checking against 0x0, but no fill should ever set that.

Reply:
Ah true good call

Reply:
Should this check even be in `getApproximateRefundsForBlockRange`? Would this not filter out valid deposits with 0x0 as the output token?

Reply:
@bmzig my thinking was that in `getApproximateRefundsForBlockRange` like in this `loadDataFromScratch` function, there is a possibility to match a fill with an `outputToken = 0x0` with a deposit. 

So that's why this check exists there now. WDYT?

Reply:
Yep. That makes sense. I think we do want this here, then.